### DISCUSSION TITLE
Development target

### DISCUSSION THREAD

**Discussion by girotobial** [Ortho4XP/Ortho4XP#2] [Q&A] [answered]
Should we be developing against the old master, or the work done on progressive_zl?

**Comment by artesim-zz** [comment, marked as answer]
I was not very far from something mergeable, but it's been about two years since I last worked on it.
I'd need to review my own code before merging it, and this feature should not impede work on the dev branch.
So I think I'll just open a Pull Request to merge feature/progressive_zl into dev, and merge it when we're sure it's ready.

**Reply by artesim-zz** [reply]
Ah ok, I see now, so this section works a lot like stack overflow. This could be great for development-related discussions, great idea !

**Reply by artesim-zz** [reply]
Ok, now that it's merged, I'll auto-mark my own answer as answer, if you don't mind :)

**Reply by girotobial** [reply]
Cheeky 😂
